src/app/containers/ATIAnalytics/canonical/index.jsx

```diff
@@ -1,19 +1,24 @@
-import React, { useEffect, useState } from 'react';
+/* eslint react/no-danger: 0 */
+import React, { useContext, useEffect, useState } from 'react';
 import { string } from 'prop-types';
 import { atiBaseUrl } from '../atiUrl';
 import sendBeacon from '../../../lib/analyticsUtils/sendBeacon';
+import { RequestContext } from '../../../contexts/RequestContext';
 
 const CanonicalATIAnalytics = ({ pageviewParams }) => {
-  const [atiPageViewUrl] = useState(atiBaseUrl + pageviewParams);
+  const { env } = useContext(RequestContext);
+  const [atiPageViewUrl] = useState(atiBaseUrl(env) + pageviewParams);
 
   useEffect(() => {
     sendBeacon(atiPageViewUrl);
   }, [atiPageViewUrl]);
 
   return (
-    <noscript>
-      <img height="1px" width="1px" alt="" src={atiPageViewUrl} />
-    </noscript>
+    <noscript
```

Comment:
Howcome the switch to dangerouslyinnerset?

Reply:
`noscript` was being rendered with no `innerHTML` previously.

Reply:
Have fixed this.